code/game/objects/items/weapons/mop.dm

```diff
@@ -1,3 +1,5 @@
+#define is_cleanable(A) (istype(A, /obj/effect/decal/cleanable) || istype(A, /obj/effect/rune))

```

Comment:
@tigercat2000 super super late to the party here but I believe you'll appreciate it, this being a define is MUCH better than it being a proc.

```#define``` replace all cases on the left (```is_cleanable(A)```) with the code on the right (```(istype(A, /obj/effect/decal/cleanable) || istype(A, /obj/effect/rune))```) at compile time, meaning when the game is running, the code on the right is called, IDENTICAL to if you'd hand written it, except it's a define meaning you can change them all later.
if this were a proc, it'd actually have some slowdown and overhead to it, which the ```#define``` avoids.